keystrokes/src/components/Home/Home.js

```diff
@@ -1,16 +1,57 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import Game from '../Game/Game';
 import Nav from '../NavBar/NavBar';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
+import { ref } from '../../config/constants';
+import CharacterList from '../Character/CharacterList';
+
+function Home(props) {
+	const userId = props.user.email.replace(/[.$#[\]]/g, '');
```

Comment:
what's this regex for?

Reply:
umm... i know I think i just used it to take someones email and remove the special characters to make it an id... dont remember why i felt the need to do it.